src/MTGCore/HelperClasses/ManaConvert.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MTGCore.HelperClasses
+{
+    public static class ManaConvert
+    {
+        public static string String (string manaCost)
+        {
+            string[] sections = manaCost.Split(new string[] { "{", "}" }, StringSplitOptions.RemoveEmptyEntries);
+            string output = "";
+            foreach (string section in sections)
+            {
+                output += string.Format("<img src=\"/images/{0}.svg\" height=\"20\" width=\"20\" />", section);
+            }
+            return output;
+        }
+    }
+}
```

Comment:
add a unit test to my unit test project to test this.

heres the testing documentation
http://docs.shouldly-lib.net/